test/python/cross-stack-references/__snapshots__/test.ts.snap

```diff
@@ -0,0 +1,57 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`python cross stack references synth generates JSON 1`] = `
+"{
+  \\"version\\": \\"stubbed\\",
+  \\"stacks\\": {
+    \\"source\\": {
+      \\"name\\": \\"source\\",
+      \\"constructPath\\": \\"source\\",
+      \\"workingDirectory\\": \\"stacks/source\\",
+      \\"synthesizedStackPath\\": \\"stacks/source/cdk.tf.json\\",
+      \\"annotations\\": [],
+      \\"dependencies\\": []
+    },
+    \\"passthrough\\": {
+      \\"name\\": \\"passthrough\\",
+      \\"constructPath\\": \\"passthrough\\",
+      \\"workingDirectory\\": \\"stacks/passthrough\\",
+      \\"synthesizedStackPath\\": \\"stacks/passthrough/cdk.tf.json\\",
+      \\"annotations\\": [],
+      \\"dependencies\\": [
+        \\"source\\"
+      ]
+    },
+    \\"sink\\": {
+      \\"name\\": \\"sink\\",
+      \\"constructPath\\": \\"sink\\",
+      \\"workingDirectory\\": \\"stacks/sink\\",
+      \\"synthesizedStackPath\\": \\"stacks/sink/cdk.tf.json\\",
+      \\"annotations\\": [],
+      \\"dependencies\\": [
+        \\"source\\"
+      ]
+    },
+    \\"fns\\": {
+      \\"name\\": \\"fns\\",
+      \\"constructPath\\": \\"fns\\",
+      \\"workingDirectory\\": \\"stacks/fns\\",
+      \\"synthesizedStackPath\\": \\"stacks/fns/cdk.tf.json\\",
+      \\"annotations\\": [],
+      \\"dependencies\\": [
+        \\"source\\"
+      ]
+    },
+    \\"functionOutput\\": {
+      \\"name\\": \\"functionOutput\\",
+      \\"constructPath\\": \\"functionOutput\\",
+      \\"workingDirectory\\": \\"stacks/functionOutput\\",
+      \\"synthesizedStackPath\\": \\"stacks/functionOutput/cdk.tf.json\\",
+      \\"annotations\\": [],
+      \\"dependencies\\": [
+        \\"source\\"
```

Comment:
Shouldn't `functionOutput` depend on `fns`?

Reply:
Apparently it does not need to (because the Terraform function called in `fns` returns the same result when called in `functionOutput`). Doesn't look like expected however – maybe use an instantiated resource in `fns` instead of the TF functions?

Reply:
The origin of the data is pinned down. I'd argue that this is correct, we should only automatically create data-level dependencies between stacks, but I'll make sure to document this.